websites/P/PayPal/dist/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.6",
+  "author": {
+    "name": "レム",
+    "id": "225657353696051200"
+  },
+  "contributors": [
+    {
+      "name": "Senko Nyan",
+      "id": "251098700259131404"
+    }
+  ],
+  "service": "PayPal",
+  "description": {
+    "en": "PayPal is a online payment service."
+  },
+  "url": "www.paypal.com",
```

Comment:
PayPal use many more URLs than just this, think about supporting a wider range of (sub)domains